apk-tools.yaml

```diff
@@ -1,13 +1,14 @@
 package:
   name: apk-tools
   version: 2.14.4
-  epoch: 2
-  description: "apk-tools (Wolfi package manager)"
+  epoch: 3
+  description: apk-tools (Wolfi package manager)
```

Comment:
You'll want to do `yml.preserve_quotes = True` (on https://gist.github.com/smoser/7da4ed38fb06e80d199ce28e07c20d1b) in order to avoid these bogus changes.